Rakefile

```diff
@@ -16,6 +16,7 @@ PuppetLint.configuration.send('disable_140chars')
 PuppetLint.configuration.send('disable_class_inherits_from_params_class')
 PuppetLint.configuration.send('disable_documentation')
 PuppetLint.configuration.send('disable_single_quote_string_with_variables')
+PuppetLint.configuration.send('disable_quoted_booleans')
```

Comment:
This file is managed by modulesync and will be rewritten.

Reply:
Fixed.